plugins/backstage-plugin-env0/src/components/env0-deployment-table/env0-deployment-table.tsx

```diff
@@ -18,9 +17,10 @@ import Tooltip from '@mui/material/Tooltip';
 
 const getFormattedDeploymentDuration = (deployment: Deployment) => {
   if (!deployment.finishedAt || !deployment.startedAt) return '-';
-  const durationInSeconds = dayjs
-    .duration(dayjs(deployment.finishedAt).diff(deployment.startedAt))
-    .asSeconds();
+  const startedAt = new Date(deployment.startedAt);
+  const finishedAt = new Date(deployment.finishedAt);
+
+  const durationInSeconds = (finishedAt.getTime() - startedAt.getTime()) / 1000;
```

Comment:
This isn't the same 👀 
Are we cool with it? it will be showing worse outputs here

Reply:
As discussed, it mimics the same behavior as `dayjs.asSeconds()` so we're good to go